src/.env.example

```diff
@@ -5,8 +5,11 @@ API_KEY=<your_secret_key>
 API_SECRET=<your_secret_password>
 API_URL=https://example.com/v5.3/api/
 API_LIMIT=500
+API_MODE=SharedInstance
+#API_MODE=YearSpecific
+SCHOOL_YEAR=2021,2022
```

Comment:
We should update the readme file to explain the new settings.

Reply:
Ran some tests and everything looks good. I'll approve it once you update the readme.